xapian-applications/omega/omindexcheck.cc

```diff
@@ -80,6 +80,46 @@ index_test()
     tests.insert({"img/scan-page.png", {"Zannual", "Zfed", "Zreturn",
 		  "Zwhile"}});
 #endif
+#if defined HAVE_LIBARCHIVE
+    // blank file
+    tests.insert({"odf/blank.odt", {}});
+    // corrupted file (ODP)
+    // "no text extracted from document body, but indexing metadata anyway"
+    // error on searching for any term - term not found
+    // tests.insert({"corrupt_file.odp", {"ZSnatur"}});
```

Comment:
Not sure if these comments should be here. I know they are going to be relevant in the near future if some modification are applied, but at the moment the doesn't add any relevant information.

Reply:
I have removed the comments (except those describing the file - blank, corrupt, ODF, ApacheOffice)